### DISCUSSION TITLE
GitHub student devloper pack

### DISCUSSION THREAD

**Discussion by TejashMalviya** [community/community#130446] [GitHub Education] [answered]
I applied for student devloper pack 3 days ago . Still I don't get the pack..

Only show ,
You're request is approved and benifits will be available soon..

**Comment by Pavith19** [comment]
If you applied for the GitHub Student Developer Pack and your request is approved but you haven't received the benefits yet, there are a few steps you can take to follow up and potentially expedite the process:

Steps to Follow Up
Check Your Email:

Ensure that you have received a confirmation email from GitHub about your application. Sometimes, additional instructions or updates are provided via email.
Verify Your GitHub Education Status:

Go to the GitHub Education page.
Log in with your GitHub account.
Check if your status is updated to "Student" and whether the benefits are listed as active.
Wait a Little Longer:

Although the process usually takes a few days, it can sometimes take up to a week for the benefits to be activated. Waiting a few more days might be necessary.
Contact GitHub Support:

If it's been longer than a week, contact GitHub Support for assistance.
You can open a support ticket by going to GitHub Support.

**Comment by davevad93** [comment]
Hi @TejashMalviya ,

When the benefits are available, the status label will turn purple and display the following message:

`Congrats! Your request for @TejashMalviya was approved and your benefits are now available`

It usually takes a few days, but may take longer depending on different factors such as the volume of applications.

**Comment by TejashMalviya** [comment]
My request is approved so there are probability to get benifits or not ?

**Reply by davevad93** [reply]
Yes @TejashMalviya , GitHub has approved your request, which means that in a few days your benefits will be available. 👍

**Comment by michelmerheb** [comment, marked as answer]
DId you get the benefits?

**Reply by TejashMalviya** [reply]
yes

**Comment by Robin-Sarwara** [comment]
I applied for the GitHub Student Developer Pack three days ago, and my request was approved. However, despite the approval, I have not yet received access to the benefits. My account still displays the message: "Your request was approved, and your benefits will be available soon."

Since the approval, I have tried logging out and back in, refreshing the page, and clearing my browser cache, but the issue persists. Could someone assist me in resolving this delay?

Update: I got the developer pack

**Reply by michelmerheb** [reply]
Congrats! I remember it took time for me also
